iavl/iavl_proof.go

```diff
@@ -17,20 +18,28 @@ type IAVLProof struct {
 	RootHash   []byte
 }
 
-func (proof *IAVLProof) Verify(key []byte, value []byte, root []byte) bool {
+func (proof *IAVLProof) Verify(key []byte, value []byte, root []byte, version int) bool {
```

Comment:
Interesting, I'll go check. I'd prefer the code to appear to be identical between the two, so people can easily switch back and forth just by changing the initialization.